### DISCUSSION TITLE
How to show content based on my database data ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [awcodes/filament-curator#402] [Q&A]
I have installed awcodes/filament-curator 3.3.1 in laravel 10 / filamentphp 3 app 
and filled content of some pages in editor.
But all this text is static

Also I addded filament-fabricator 2.0.4 package and use images in my cms editor.

I use glider when I need to show image from media library :

`   <x-curator-glider :media="$image" class="object-cover w-auto" glide="" fallback="" media-id="image"  />
`
But I need on frontend part to show some html content based on my database data, like some macros inside of blocks of content...

Is there a way to make this ?

**Comment by awcodes** [comment]
Sorry, I'm not understanding what you are asking about.

**Reply by sergeynilov** [reply]
Some hooks when blocks of data are rendered on frontend , where I could insert piecies of html code, which I render based on data from db

**Reply by awcodes** [reply]
This sounds like it might be more of question for Fabricator and not Curator.
